tsdb/index/index.go

```diff
@@ -744,14 +729,18 @@ func (w *Writer) writePostingsOffsetTable() error {
 	}
 	w.fPO = nil
 
+	return w.writeLenghtAndHash(startPos)
+}
+
+func (w *Writer) writeLenghtAndHash(startPos uint64) error {
 	// Write out the length.
 	w.buf1.Reset()
 	w.buf1.PutBE32int(int(w.f.pos - startPos - 4))
 	if err := w.writeAt(w.buf1.Get(), startPos); err != nil {
 		return err
 	}
 
-	// Finally write the hash.
+	// write hash
```

Comment:
```suggestion
	// Write out the hash.
```

See: https://thanos.io/tip/contributing/coding-style-guide.md/#comments-should-be-full-sentences

Reply:
fixed